portal/views/patients.py

```diff
@@ -51,16 +52,24 @@ def patients_root():
     consented_users = [u.user_id for u in consent_query]
 
     if user.has_role(ROLE.STAFF):
-        request_org_list = request.args.get('org_list', None)
+        pref_org_list = None
+        # check user table preference for organization filters
+        pref = TablePreference.query.filter_by(table_name='patientList',
+                                               user_id=user.id).first()
+        if pref:
+            pref_data = pref.as_json()
```

Comment:
This could be done without converting the pref to json, by just pulling the attributes from `pref`. For example:
```
if pref and pref.filters:
   pref_org_list = pref.filters.get('orgs_filter_control') # will return None if 'orgs_filter_control' isn't in pref.filters
```